javascript/apps/taiga/src/assets/i18n/comments/en-US.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "Comments",
+  "leave_comment": "Leave a comment",
```

Comment:
The dots are missing
![image](https://github.com/taigaio/taiga/assets/5970959/59ff5ee2-1378-4ab5-bdcf-2054699b7998)